db/migrate/20151111165020_rename_miq_search_db.rb

```diff
@@ -6,10 +10,11 @@ class RenameMiqSearchDb < ActiveRecord::Migration
     VmCloud       ManageIQ::Providers::CloudManager::Vm
   )]
 
-  def change
```

Comment:
Good find.  What does this do in a down?  I'm not sure how it would know how to reverse this.